.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,10 @@
+---
+name: 💬 Question
+about: If you need help understanding HumanConnection.
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-03T12:20:22Z" title="Wednesday, April 3rd 2019, 2:20:22 pm +02:00">Apr 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,6 +1,6 @@
 ---
 name: 💬 Question
-about: If you need help understanding the HumanConnection.
+about: If you need help understanding HumanConnection.
```


@ulfgebhardt sorry for nitpicking: You forgot to add a space. @Hardy-Peaceglobal already told be about how to correctly spell it:

Wrong:
"HumanConnection"
"Human-Connection"
"Human connection"
"HC"

Correct:
**"Human Connection"**